README.md

```diff
@@ -1,10 +1,11 @@
+[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://knative.slack.com/archives/C9JP909F0)
```

Comment:
I wonder if we should have this point to _slack.knative.dev_ instead? I mean first target audience for this should be people who are NOT in slack already and dunno how to find us, I always feel that pointing to the non-invitation link is a barrier (because there're zero instructions on how the hell can I get invited on that "sign in" page). not sure what you think @n3wscott

Reply:
the goal of that link is to get someone from github to the #eventing channel

Reply:
@n3wscott understood, but there are two cases, either he's in already, i.e. he knows his way around. Or he never joined, and the sing-in page will be a gatekeeper for him to join. Does that make sense?

Reply:
I'm fine if you wanna take this off scope of this PR, just wanted to float this idea around.

Reply:
this PR changes the original link from the default channel in slack to the eventing room. So maybe what you are asking for is a signup badge? let me work that up